docs/limits.md

```diff
@@ -7,12 +7,12 @@ sidebar_class_name: sidebar-item-limits
 
 This page outlines our default service limits at Momento. Note that limits can be adjusted as needed, please reach out to hello@momentohq.com with your requests. Self service coming soon!
 
-| Limit                | Value              |
-|----------------------|--------------------|
-| Item Size            | 1MB                |
-| Max Cache Count      | 10                 |
-| Data Plane RPS*      | 100 RPS* Per Cache |
-| Data Plane Bandwidth | 1MB/s Per Cache    |
-| Control Plane RPS*   | 20 RPS             |
+| Limit                                 | Value    |
+|---------------------------------------|----------|
+| API Rate per cache (Data Plane)       | 100 RPS* |
+| Throughput per cache                  | 1MB/s    |
+| Item Size                             | 1MB      |
+| API Rate per customer (Control Plane) | 5 RPS    |
+| Max Cache Count                       | 10       |
 
-_*RPS = Requests/Second_
+*RPS = Requests/Second
```

Comment:
I think something off with footnote in markdown here @mjindalmotiv8 were you trying to keep italics or bold this?

Reply:
I can make it italics